include/zephyr/drivers/sensor.h

```diff
@@ -640,7 +640,7 @@ static inline void sensor_g_to_ms2(int32_t g, struct sensor_value *ms2)
  */
 static inline int32_t sensor_ms2_to_ug(const struct sensor_value *ms2)
 {
-	int64_t micro_ms2 = ms2->val1 * 1000000LL + ms2->val2;
+	int64_t micro_ms2 = (ms2->val1 * INT64_C(1000000)) + ms2->val2;
```

Comment:
Is this change related to `test: verify accel/gyro data for icm42688`?

Reply:
Yes, `LL` wasn't casting correctly and I wanted to be more explicit on both the type and the order of operations.